tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -5920,8 +5929,18 @@ node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(483,
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(500,22): error TS2339: Property 'createChild' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeWrapper.js(18,22): error TS2339: Property 'singleton' does not exist on type 'Window'.
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeWrapper.js(33,24): error TS2694: Namespace 'Protocol' has no exported member 'Page'.
+node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeWrapper.js(38,30): error TS2351: Cannot use 'new' with an expression whose type lacks a call or construct signature.
```

Comment:
this reference to DeviceModeView is borked too. I think it might be a change to the valueDeclaration?